tests/conftest.py

```diff
@@ -903,40 +902,8 @@ def is_port_in_use(host: str, port: int) -> bool:
         return s.connect_ex((host, port)) == 0
 
 
-class BaseP2PLibp2pTest:
-    """Base class for p2p libp2p tests"""
-
-    cwd: str
-    t: str
-    tmp_dir: str
-    log_files: List[str] = []
-    multiplexers: List[Multiplexer] = []
-    capture_log = True
-
-    @classmethod
-    def setup_class(cls):
-        """Set the test up"""
-        cls.cwd, cls.t = os.getcwd(), tempfile.mkdtemp()
-        cls.tmp_dir = os.path.join(cls.t, "temp_dir")
-        Path(cls.tmp_dir).mkdir(exist_ok=True)
-        os.chdir(cls.t)
-        atexit.register(cls.teardown_class)
-
-    @classmethod
-    def teardown_class(cls):
-        """Tear down the test"""
-        logger.debug(f"Cleaning up {cls.__name__}")
-        for mux in cls.multiplexers:
-            mux.disconnect()
-        cls.multiplexers.clear()
-        cls.log_files.clear()
-        os.chdir(cls.cwd)
-        remove_test_directory(cls.t)
-        logger.debug(f"Teardown of {cls.__name__} successful")
-
-
 def _make_libp2p_connection(
-    data_dir: str,
+    data_dir: str = TEMP_LIBP2P_TEST_DIR,
     port: Optional[int] = None,
     host: Optional[str] = DEFAULT_HOST,
     relay: bool = True,
```

Comment:
In response to feedback in #230](https://github.com/valory-xyz/open-aea/pull/230#discussion_r926450743): `Optional` is now needed 

```
tests/conftest.py:908: error: Incompatible default for argument "host" (default has type "Optional[str]", argument has type "str")
tests/conftest.py:914: error: Incompatible default for argument "delegate_host" (default has type "Optional[str]", argument has type "str")
tests/conftest.py:916: error: Incompatible default for argument "mailbox_host" (default has type "Optional[str]", argument has type "str")
tests/conftest.py:990: error: Incompatible default for argument "node_host" (default has type "Optional[str]", argument has type "str")
tests/conftest.py:1039: error: Incompatible default for argument "node_host" (default has type "Optional[str]", argument has type "str")
```